Sources/AdminPanelProvider/Routes/LoginRoutes.swift

```diff
@@ -4,15 +4,17 @@ import AuthProvider
 
 public final class LoginRoutes: RouteCollection {
     public let controller: LoginController
-    public let ssoController: SSO?
 
-    public init(renderer: ViewRenderer, ssoController: SSO?, mailgun: Mailgun?, panelConfig: PanelConfig) {
+    public init(
```

Comment:
Any reason not to have just one argument for loginController instead of passing on the dependencies?

Reply:
Good question, I haven't really considered why it was done this way, but no I can't think of any. Updating it - thanks!